base/linalg/lu.jl

```diff
@@ -132,6 +141,13 @@ lufact(x::Number) = LU(fill(x, 1, 1), BlasInt[1], x == 0 ? one(BlasInt) : zero(B
 lufact(F::LU) = F
 
 lu(x::Number) = (one(x), x, 1)
+
+"""
+    lu(A) -> L, U, p
```

Comment:
should `pivot` be documented or is that only intended for going through `lufact` going forward?